githubkit/auth/oauth.py

```diff
@@ -29,7 +29,7 @@
 
 
 def create_device_code(
-    github: "GitHubCore", client_id: str, scopes: Optional[list[str]] = None
+    github: "GitHubCore", client_id: str, scopes: Optional[Iterable[str]] = None
```

Comment:
`Iterable` does not have length func. This will make the code bellow `if scopes:` becomes `True` when the iterable does not have values.

Reply:
Darn, I missed that, good catch!